compiler/src/parsing/well_formedness.re

```diff
@@ -305,24 +247,36 @@ let no_letrec_mut = (errs, super) => {
   };
 };
 
+let string_is_all_zeros = s => {
+  String.for_all(c => c == '0', s);
```

Comment:
Good catch. There's still a small bug here—we support underscores, so we'll need to handle `0_0` as well.

Reply:
Updated 🎉